fuzzers/klee/builder.Dockerfile

```diff
@@ -24,8 +24,7 @@ RUN apt-get update -y && \
 # Install KLEE dependencies.
 RUN apt-get install -y \
     cmake-data build-essential curl libcap-dev \
-    git cmake libncurses5-dev python-minimal \
```

Comment:
Why are we making this change?

Reply:
So, this installs python2 on top of already existing python3. Now pip install does not support python2 anymore, so pip upgrade in command above upgrades to pip 21.3.4, which then fails with

```
#13 [ 9/14] RUN pip install wllvm
#13 sha256:bc65c9f0280549ac4d0c50639cdb9234c10d5b9f7b1a7fd509a6310a4c96881b
#13 0.290 Traceback (most recent call last):
#13 0.290   File "/usr/local/bin/pip", line 7, in <module>
#13 0.290     from pip._internal.cli.main import main
#13 0.290   File "/usr/local/lib/python2.7/dist-packages/pip/_internal/cli/main.py", line 60
#13 0.291     sys.stderr.write(f"ERROR: {exc}")
#13 0.291                                    ^
#13 0.292 SyntaxError: invalid syntax
#13 ERROR: executor failed running [/bin/sh -c pip install wllvm]: exit code: 1
```

There is no point of leaving python2 there, and to fix exception, just install wllvm using pip3.